pkgs/development/libraries/mesa/default.nix

```diff
@@ -27,7 +27,9 @@
 with stdenv.lib;
 
 let
-  version = "20.0.2";
+  # Release calendar: https://www.mesa3d.org/release-calendar.html
+  # Release frequency: https://www.mesa3d.org/releasing.html#schedule
+  version = "20.0.7"; # Update only to the final (last planned) release (i.e. X.Y.MAX)?
```

Comment:
@vcunat what do you think of the idea to only update to the final (last planned) release of a X.Y series?

This isn't ideal, but since we currently lack the capacity to update `mesa` [~every two weeks](https://www.mesa3d.org/releasing.html#schedule), our channels are usually delayed by a few days anyway, and this causes lots of rebuilds that sounded like an idea that might be worth a try. My hope is that the last/final release a a series would be the most bug free / well tested.
But of course this wouldn't have to be a strict rule (sometimes we might e.g. require an update to fix other packages that run into bugs or require new features).

New features releases (new X.Y series) are available [approx. every 3 months](https://www.mesa3d.org/releasing.html#schedule) so that wouldn't be a problem for us, but it's unfortunately also a bit long.

Reply:
For the non-feature bumps I expect we could go without manual testing (into staging), in which case the update is quite easy to do.  I also had some ideas about reducing the rebuild amounts significantly, at the very least on non-feature bumps (I could try to recover the ideas).  But perhaps there's no real demand for frequent updates anyway?

Reply:
Ah, the rebuild ideas are here: https://github.com/NixOS/nixpkgs/issues/44831